docs/getting_started/benchmark_owner_demo.md

```diff
@@ -74,124 +74,122 @@ medperf test run \
    --labels_path "medperf_tutorial/mock_chexpert/labels"
 ```
 
-Assuming the test passes, we are ready to submit the MLCubes to the MedPerf server.
+Assuming the test passes successfully, you are ready to submit the MLCubes to the MedPerf server.
 
 ## 4. Submitting the MLCubes
 
 #### How does MedPerf Recognize an MLCube?
 
 {% include "getting_started/shared/mlcube_submission_overview.md" %}
 
-To prepare the files of our three MLCubes, run (make sure you are in MedPerf's root folder):
+To prepare the files of our three MLCubes, run the following command ensuring you are in MedPerf's root folder:
 
 ```bash
 python scripts/package-mlcube.py --mlcube medperf_tutorial/chexpert_prep/mlcube --mlcube-types data-preparator
 python scripts/package-mlcube.py --mlcube medperf_tutorial/xrv_densenet/mlcube --mlcube-types model
 python scripts/package-mlcube.py --mlcube medperf_tutorial/metrics/mlcube --mlcube-types metrics
 ```
 
-For each MLCube, this script will create a new folder in the MLCube directory, named `assets`, containing all the files that should be hosted separately.
+For each MLCube, this script will create a new folder named `deploy` in the MLCube directory. This folder will contain all the files that should be hosted separately.
 
 {% include "getting_started/shared/redirect_to_hosting_files.md" %}
 
 #### Submit the MLCubes
 
-To submit the MLCubes:
+#### Data Preparator MLCube
 
-1. The Data Preparator MLCube:
+For the Data Preparator MLCube, the submission should include:
 
-    - The submission should include:
+- The URL to the hosted mlcube manifest file, which is:
 
-        a. The URL to the hosted mlcube manifest file:
+    ```text
+    {{ prep_mlcube }}
 
-        ```text
-        {{ prep_mlcube }}
+    ```
 
-        ```
+- The URL to the hosted mlcube parameters file, which is:
 
-        b. The URL to the hosted mlcube parameters file:
+    ```text
+    {{ prep_params }}
+    ```
 
-        ```text
-        {{ prep_params }}
-        ```
+- The command to submit, as presented below:
 
-    - The command to submit:
+    ```bash
+    medperf mlcube submit \
+        --name my-prep-cube \
+        --mlcube-file "{{ prep_mlcube }}" \
+        --parameters-file "{{ prep_params }}" \
+    ```
 
```

Comment:
This should not be listed as a point under "what to include in the submission". See the two MLCubes below, it should be written as follows:

```
Use the following command to submit:
...
```